posixlib/src/main/scala/scala/scalanative/runtime/SocketHelpers.scala

```diff
@@ -19,17 +19,14 @@ object SocketHelpers {
 
   def isReachableByEcho(ip: String, timeout: Int, port: Int): Boolean = {
```

Comment:
Could this and the following one be done on the stack as well to be faster?

Reply:
I really like the idea of doing what you can on the stack is better and more C like. Since the other place has a good example I think it might be a good idea to do it the same here as well. It is up to you but I think it would be nice. You being a former C programmer have to like the idea, right?